deepreg/model/backbone/u_net.py

```diff
@@ -243,10 +253,39 @@ def build_skip_block(self) -> Union[tf.keras.Model, tfkl.Layer]:
         else:
             return tfkl.Add()
 
+    def build_decode_conv_block(
```

Comment:
As w/ previous comment; this doesn't belong with this PR. This is totally unrelated (unless I'm missing something...)

Reply:
same as previous, it's gone ^^